templates/example_config.json

```
@@ -0,0 +1,3 @@
+{
+  "example_key" : "Hello world!"
+}
```

```suggestion
}

```

So that github doesn't complain (and linters in the future).